src/components/RequireAuth.tsx

```
@@ -0,0 +1,81 @@
+                email: payload.email,
+                role: payload.role,
+                exp: payload.exp
+              })
+            } else {
+              // Token is expired
+              console.log('Token expired')
+              reset()
+            }
+          }
+        } catch (error) {
+          console.error('Error validating token:', error)
```

ESLint

[error] Unexpected console statement.

(no-console)